src/views/Home/index.tsx

```diff
@@ -13,21 +12,19 @@ import { fetchSystemIntakes } from 'types/routines';
 import { SystemIntakeForm } from 'types/systemIntake';
 import './index.scss';
 
-type HomeProps = RouteComponentProps & {
-  auth: any;
-};
+type HomeProps = RouteComponentProps;
 
-const Home = ({ auth, history }: HomeProps) => {
-  const [isAuthenticated] = useAuth(auth);
+const Home = ({ history }: HomeProps) => {
+  const { authState } = useOktaAuth();
   const dispatch = useDispatch();
   const systemIntakes = useSelector(
     (state: AppState) => state.systemIntakes.systemIntakes
   );
   useEffect(() => {
-    if (isAuthenticated) {
+    if (authState.isAuthenticated) {
       dispatch(fetchSystemIntakes());
     }
-  }, [dispatch, isAuthenticated]);
+  }, [dispatch, authState.isAuthenticated]);
```

Comment:
I think we should leave this for now. I'm not totally sure if we need the dependencies, but since `isAuthenticated` can change, I want to leave it until we know for sure.

I've had empty array in the effect hooks come back and bite me a few times so I only use them when I know it only happens on mount and never again.